fe/fe-core/src/test/java/org/apache/doris/load/TabletLoadInfoTest.java

```diff
@@ -60,10 +60,10 @@ public void testSerialization() throws Exception {
 
         Assert.assertEquals("hdfs://host:port/dir", tabletLoadInfo1.getFilePath());
         Assert.assertEquals(1L, tabletLoadInfo1.getFileSize());
-
-        Assert.assertTrue(tabletLoadInfo1.equals(tabletLoadInfo));
-        Assert.assertTrue(rTabletLoadInfo0.equals(tabletLoadInfo0));
-        Assert.assertFalse(rTabletLoadInfo0.equals(tabletLoadInfo1));
+        
```

Comment:
redundant blank.

Reply:
It will make build failed

Reply:
![image](https://user-images.githubusercontent.com/30525741/209507505-f09d0cc6-b2c6-4c81-93a2-6a9addca8fc1.png)